### PR TITLE
dev/reference/tools: update version limit for DM/Syncer

### PR DESCRIPTION
**Pull request by yikeke**
Via: https://github.com/pingcap/docs-cn/pull/1312/ PTAL @lilin90 @csuzhangxc

### CODE REVIEW

File: dev/reference/tools/data-migration/overview.md [hunk 1]
```diff
@@ -78,6 +78,12 @@ Before using the DM tool, note the following restrictions:
     - 5.5 < MySQL version < 5.8
     - MariaDB version >= 10.1.2
 
+    > **Note**:
```
Comment (CaitinChen) [inline dev/reference/tools/data-migration/overview.md]:
```suggestion
    > **Note:**
```

File: dev/reference/tools/syncer.md [hunk 2]
```diff
@@ -356,6 +356,12 @@ Before replicating data using Syncer, check the following items:
 
         In earlier versions of MariaDB, the format of some binlog field types is inconsistent with that in MySQL.
 
+    > **Note**:
```
Comment (CaitinChen) [inline dev/reference/tools/syncer.md]:
```suggestion
    > **Note:**
```

Comment (yikeke) [reply dev/reference/tools/syncer.md]: PTAL again, thanks!

**Comment by lilin90** [comment]
@csuzhangxc PTAL

File: dev/reference/tools/data-migration/overview.md [hunk 3]
```diff
@@ -78,6 +78,12 @@ Before using the DM tool, note the following restrictions:
     - 5.5 < MySQL version < 5.8
     - MariaDB version >= 10.1.2
 
+    > **Note:** 
+    >
+    > If there is a master-slave replication structure between the upstream MySQL/MariaDB servers, then choose the following version.
+    > - 5.7.1 < MySQL version < 5.8
+    > - MariaDB version >= 10.1.3
```
Comment (csuzhangxc) [inline dev/reference/tools/data-migration/overview.md]: this list seems not to work on the website, change to another format? and can you help me to fix it in docs-cn (https://pingcap.com/docs-cn/tools/dm/overview/#%E4%BD%BF%E7%94%A8%E9%99%90%E5%88%B6)?

Comment (yikeke) [reply dev/reference/tools/data-migration/overview.md]: Okay.

**Pull request merged**